README.md

```diff
@@ -53,6 +53,7 @@ Any of these can be passed into the options object for each function.
 | length                   | Integer, length of password.                        |       10      |
 | numbers                  | Boolean, put numbers in password.                   |     false     |
 | symbols                  | Boolean, put symbols in password.                   |     false     |
+| excludes				 				 | String, characters to be excluded from password.		 |      null  	 |
```

Comment:
Could you drop this below `excludeSimilarCharacters` and change the spacing from tabs to spaces?